src/main/resources/log4j.properties

```diff
@@ -0,0 +1,23 @@
+log4j.rootCategory=INFO, CONSOLE
+
+PID=????
+LOG_LEVEL_PATTERN=%5p
+LOG_PATTERN=[%d{yyyy-MM-dd HH:mm:ss.SSS}] %x boot%X{context} - ${PID} ${LOG_LEVEL_PATTERN} [%t] --- %c{1}: %m%n
+
+# CONSOLE is set to be a ConsoleAppender using a PatternLayout.
+log4j.appender.CONSOLE=org.apache.log4j.ConsoleAppender
+log4j.appender.CONSOLE.layout=org.apache.log4j.PatternLayout
```

Comment:
I don't know if it really matters, but I was reading the docs and it seems that there are suggestions to use the `EnhancedPatternLayout` but I'm not sure why https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/PatternLayout.html